README.md

```diff
@@ -152,31 +150,37 @@ add_action('Flynt/afterRegisterComponents', function () {
                 'type' => 'flexible_content',
                 'button_label' => 'Add Component',
                 'layouts' => [
-                    Api::loadFields('BlockWysiwyg', 'layout'),
-                ],
-            ],
+                    Components\BlockWysiwyg\getACFLayout(),
+                ] 
+            ]
         ],
         'location' => [
             [
                 [
                     'param' => 'post_type',
                     'operator' => '==',
-                    'value' => 'page',
+                    'value' => 'page'
                 ],
-            ],
-        ],
+                [
+                    'param' => 'page_type',
+                    'operator' => '!=',
+                    'value' => 'posts_page'
```

Comment:
What do you think about removing the whole location part, just to keep it short? Or replace it with [...] or something?